lib/IonRequest.ts

```diff
@@ -324,13 +324,10 @@ export default class IonRequest {
   }
 
   private static validateDidSuffix (didSuffix: string) {
-    if (didSuffix.length !== 46) {
-      throw new IonError(ErrorCode.DidSuffixIncorrectLength, 'DID suffix must be 46 bytes.');
-    }
-
     if (!Encoder.isBase64UrlString(didSuffix)) {
       throw new IonError(ErrorCode.DidSuffixIncorrectEncoding, 'DID suffix must be base64url string.');
     }
```

Comment:
Similar to comment earlier: not seeing the value of this check if we are already doing the same/stricter validation. In fact not seeing a lot of value of this entire method if all what we are gaining is avoiding the need to pass `'didSuffix'`. There maybe  a case to be made to keep this method, but then remove the redundant check.

Reply:
Moved the method to somewhere that makes more sense